nginx/modsecurity/ngx_http_modsecurity.c

```diff
@@ -1136,3 +1236,37 @@ ngx_http_modsecurity_drop_action(request_rec *r)
     ctx->r->connection->error = 1;
     return 0;
 }
+
+    static ngx_int_t
+ngx_http_variable_get_modsec_latency_detect_mode(ngx_http_request_t* r,
+    ngx_http_variable_value_t* v, uintptr_t data)
+{
+    return NGX_OK;
+}
+
+    static ngx_int_t
+ngx_http_variable_get_modsec_latency_prevent_mode(ngx_http_request_t* r,
+     ngx_http_variable_value_t* v, uintptr_t data)
+{
+    return NGX_OK;
+}
+
+    static ngx_int_t
+ngx_http_modsecurity_set_modsec_latency(ngx_http_request_t* r,
+    ngx_http_variable_value_t* v, ngx_msec_int_t modsec_latency)
+{
+    u_char* p;
+
+    p = ngx_pnalloc(r->pool, NGX_TIME_T_LEN + 4);
+    if (p == NULL) {
+        return NGX_ERROR;
+    }
+
+    v->len = ngx_sprintf(p, "%T.%03M", (time_t)(modsec_latency) / 1000, modsec_latency % 1000) - p;
```

Comment:
The time format as well as sprintf reference has been taken from the nginx core module where other time format fields like timeTaken and serverResponseLatency have similar time formats. Kept it the same so that we have consistency. 
https://msazure.visualstudio.com/One/_git/networking-roxyf-nginx?path=%2Fsrc%2Fhttp%2Fngx_http_upstream.c&version=GBmaster&line=5393&lineEnd=5394&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents